examples/simple/igraph_coloring.c

```diff
@@ -15,21 +15,8 @@ int main(void) {
     /* As with all igraph functions, the vector in which the result is returned must
        be initialized in advance. */
     igraph_vector_int_init(&colors, 0);
-    igraph_vertex_coloring_greedy(&graph, &colors, IGRAPH_COLORING_GREEDY_COLORED_NEIGHBORS);
 
-    /* Verify that the colouring is valid, i.e. no two adjacent vertices have the same colour. */
-    {
-        igraph_integer_t i;
-        /* Store the edge count to avoid the overhead from igraph_ecount in the for loop. */
-        igraph_integer_t no_of_edges = igraph_ecount(&graph);
-        for (i = 0; i < no_of_edges; ++i) {
-            if ( VECTOR(colors)[ IGRAPH_FROM(&graph, i) ] == VECTOR(colors)[ IGRAPH_TO(&graph, i) ]  ) {
-                printf("Inconsistent coloring! Vertices %" IGRAPH_PRId " and %" IGRAPH_PRId " are adjacent but have the same color.\n",
-                       IGRAPH_FROM(&graph, i), IGRAPH_TO(&graph, i));
-                abort();
-            }
-        }
-    }
```

Comment:
Please re-add these removed lines to the example, as they still serve an educational purpose.